go/pio/protocols/pins/pins.go

```diff
@@ -33,6 +34,7 @@ var (
 	PWR_SWITCH  gpio.PinIO   = &pin{name: "PWR_SWITCH"}  //
 	X32KFOUT    gpio.PinIO   = &pin{name: "X32KFOUT"}    // Clock output of 32Khz crystal
```

Comment:
yes, I was somewhat puzzled by that list. Perhaps everything other than the non-functional pins (power and such) is superseded by the ByName & ByFunction machinery?